Misc/NEWS.d/next/Library/2021-01-05-23-55-24.bpo-42834.LxRnZC.rst

```diff
@@ -0,0 +1,2 @@
+Convert static internal caches in the ``_json.c`` module to work with
+subinterpreters.
```

Comment:
Convert to what? I suggest a more generic phrasing:

```
Make internal caches of the :mod:`_json` module compatible with subinterpreters.
```

Reply:
Oops you're right, this was left behind from an older PR version.